packages/app/src/app/overmind/internalActions.ts

```diff
@@ -523,9 +523,7 @@ export const handleError = (
     error.message.startsWith(
       'You need to be signed in to fork a server template.'
     ) ||
-    error.message.startsWith(
-      'You need to be signed in to fork a cloud sandbox.'
-    )
+    error.message.startsWith('You need to be signed in to fork a devbox.')
```

Comment:
It is very likely we did not change this error message, and need to keep it as is or we are unable to identify the error.

Reply:
this looks like a BE error message. kept it the same on my branch